packages/wagmi/src/utils/defaultWagmiReactConfig.ts

```diff
@@ -33,7 +34,9 @@ export function defaultWagmiConfig({
   enableCoinbase,
   enableEmail,
   enableInjected,
-  auth,
+  auth = {
+    showWallets: true
```

Comment:
Is `showWallets` documented?

Reply:
Not yet

cc @glitch-txs 

You can find a summary of the added flags here -> https://www.notion.so/walletconnect/Sven-OOO-20-5-24-5-1d9b7bcdf2074d0f866b5e3de262e9b9